astropy/units/format/base.py

```diff
@@ -125,9 +119,21 @@ def _format_inline_fraction(
             return f"{scale}/ {denominator}"
         return f"{scale}{numerator} / {denominator}"
 
+    @classmethod
+    def _format_multiline_fraction(
+        cls, scale: str, numerator: str, denominator: str
+    ) -> str:
+        # By default, we just warn that we do not have a multiline format.
+        warnings.warn(
+            f"format {cls.__name__} does not support multiline "
```

Comment:
Our documentation uses the format names, not the names of the classes:
```suggestion
            f"{cls.name!r} format does not support multiline "
```

Reply:
Oops. Now done.